rules/autoconsent/aa.json

```diff
@@ -1,4 +1,5 @@
 {
+    "$schema": "https://raw.githubusercontent.com/duckduckgo/autoconsent/refs/heads/main/rules/schema.json",
```

Comment:
is it possible to point to a local file?

Reply:
Seems to work fine in my testing. It might be IDE dependent. We can do local refs for now until it’s a problem or we have a need for third party rules. 6cc349d

Reply:
Actually, hm, I’m having second thoughts. Using `”../schema.json”` as the `$schema` means that the combined rules in `rules/rules.json` have invalid `$schema`s as they’re in a different directory. It seems likely to me that these rules will move around as the project evolves and we explore remote configuration etc. Using an absolute URL ensures that rules are atomic: you can move them, copy/paste them, whatever, and they remain valid.

I reverted 6cc349d0f77da19e18f96fab6deb8769c58a7d82.